p2p/utils/error.go

```diff
@@ -0,0 +1,15 @@
+/**
+*  @file
+*  @copyright defined in go-seele/LICENSE
+ */
+package utils
+
+import ()
+
+// IsTemporaryError checks whether the given error should be considered temporary.
+func IsTemporaryError(err error) bool {
+	tempErr, ok := err.(interface {
+		Temporary() bool
+	})
+	return ok && tempErr.Temporary() || isPacketTooBig(err)
```

Comment:
PacketTooBig is temporary error?

Reply:
no use for now, I will use it and refine it later